src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -2243,5 +2228,11 @@ private static string FormatPropertyDocumentation(TypeProperty property)
 
             return null;
         }
+
+        [GeneratedRegex(@"'br:(.*?):?'?$", RegexOptions.IgnoreCase | RegexOptions.ExplicitCapture | RegexOptions.Compiled, "en-US")]
```

Comment:
>"en-US" [](http://example.com/codeflow?start=123&length=7)

Is this the equivalent of InvariantCulture in the classic Regex?

Reply:
Wait a sec...this was done by VS code fix, and I missed that the culture parameter was added. We should use `InvariantCulture` instead.

Reply:
Fixed.